Web/Models/Entities/Post.php

```diff
@@ -132,13 +132,18 @@ function toggleLike(User $user): bool
 
         foreach($this->getChildren() as $attachment) {
             if($attachment instanceof Post)
-                $attachment->setLike($liked, $user);
+                $attachment->setLikeRecurse($liked, $user, 2);
         }
 
         return $liked;
     }
 
     function setLike(bool $liked, User $user): void
+    {
+        $this->setLikeRecurse($liked, $user, 1);
+    }
+
+    private function setLikeRecurse(bool $liked, User $user, int $depth): void
```

Comment:
By convention, private functions are on the top of the class body + methods should be defined before they're mentioned in others. Also, rename method to `likeRecursively` or something like this, would you kindly?

Reply:
Done